recipes/raxml/meta.yaml

```diff
@@ -0,0 +1,15 @@
+package:
+  name: raxml
+  version: "8.2.4"
+build:
+  number: 0
+source:
+  fn: v8.2.4.tar.gz
+  url: https://github.com/stamatak/standard-RAxML/archive/v8.2.4.tar.gz
+requirements:
+  build:
+  run:
```

Comment:
We try to always have a test case that checks whether the binary is working properly.